modopt/opt/algorithms/base.py

```diff
@@ -34,6 +34,28 @@ class SetUp(Observable):
     use_gpu : bool, optional
         Option to use available GPU
 
+    Notes
+    -----
+    If provided, the ``metrics`` argument should be a nested dictionary of the
+    following form::
+
+        metrics = {
+            "metric_name": {
+                "metric": callable,
+                "mapping": {"x_new": "test"},
+                "cst_kwargs": {"ref": ref_image},
+                "early_stopping": False,
+            }
+        }
```

Comment:
Very minor point but, for the sake of code style consistency, I would recommend that you swap the `"` for `'`.